content_script.js

```diff
@@ -1,13 +1,9 @@
 var regex = /CC\"\>\[([A-Z]){1},([0-9]){1}\]/g;
 var source = document.getElementsByTagName('body')[0].innerHTML;
 
-var inputs = [
-    document.getElementsByName('message4')[0],
-    document.getElementsByName('message5')[0],
-    document.getElementsByName('message6')[0]
-];
+var inputs = $('input[name^="message"][type="password"]');
```

Comment:
Isn't it possible to use `document.querySelectorAll` with the same query and skip jQuery?